bitwarden_license/src/Portal/Views/Policies/Edit.cshtml

```diff
@@ -32,6 +32,19 @@
             @i18nService.T("OnlyOrganizationPolicyWarning")
         </div>
     }
+    
+    @if (Model.PolicyType == PolicyType.RequireSso)
+        {
```

Comment:
Mind the auto-formatting on these, you'll need to outdent this block to match the ones above/below it.

Reply:
Updated!